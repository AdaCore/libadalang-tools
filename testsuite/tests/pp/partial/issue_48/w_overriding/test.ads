package Test is
   type Foo is interface;

   procedure Bar (Self : Foo; I : Integer) is abstract;

   type Baz is new Foo with null record;

   overriding procedure Bar (Self : Baz; I : Integer);

end Test;
