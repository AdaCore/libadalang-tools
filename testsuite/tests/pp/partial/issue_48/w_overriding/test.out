*************************************
test.ads(<SubpSpec test.ads:8:15-8:54>) - 8:15-8:54
^
procedure Bar (Self : Baz; I : Integer)$
*************************************

*************************************
test.ads(<SubpDecl ["Bar"] test.ads:8:4-8:55>) - 8:1-8:55
^
   overriding procedure Bar (Self : Baz; I : Integer);$
*************************************

*************************************
test.adb(<SubpSpec test.adb:3:4-5:22>) - 3:1-5:22
^
   procedure Bar (Self : Baz; I : Integer)$
*************************************

*************************************
test.adb(<SubpBody ["Bar"] test.adb:2:4-9:12>) - 2:1-9:12
^
   overriding procedure Bar (Self : Baz; I : Integer) is
   begin
      null;
   end Bar;$
*************************************

